src/BloomExe/UrlPathString.cs

```diff
@@ -26,7 +26,17 @@ public class UrlPathString
 		public static UrlPathString CreateFromUrlEncodedString(string encoded)
```

Comment:
I'd sure rather not have this fix here; it pollutes what this class does.  Conceivably this could be done where we already have some migration code... a regex expression would be enough to fix it for a whole document.